common/src/main/java/org/apache/sedona/common/Functions.java

```diff
@@ -2377,4 +2377,57 @@ public static Geometry rotate(Geometry geometry, double angle, Geometry pointOri
     AffineTransformation rotation = AffineTransformation.rotationInstance(angle, originX, originY);
     return rotation.transform(geometry);
   }
+
+  /**
+   * This function returns the m coordinate of the closest point in a LineString to a Point. If the
+   * LineString does not have M values, it will interpolate the M value.
```

Comment:
nit: i think this is misleading saying `If the LineString does not have M values, it will interpolate the M value.`. Did you mean `If the Point doesn't have M value, it will interpolate the M value`?